seqr/views/apis/report_api.py

```diff
@@ -743,10 +754,16 @@ def gregor_export(request, consent_code):
             dict(**base_phenotype_row, **_get_phenotype_row(feature)) for feature in individual.absent_features or []
         ]
 
-        # analyte table
-        analyte_rows.append(dict(participant_id=participant_id, **_get_analyte_row(individual)))
+        analyte_id = None
+        # airtable data
+        if airtable_sample:
+            analyte_id = f'Broad_{airtable_sample[SMID_FIELD]}'
+            airtable_metadata = airtable_sample.get('metadata')
+            if airtable_metadata:
+                airtable_rows.append(dict(analyte_id=analyte_id, **airtable_metadata, **_get_experiment_ids(airtable_sample)))
 
-    airtable_rows = []  # TODO populate airtable data once new columns are confirmed
+        # analyte table
+        analyte_rows.append(dict(participant_id=participant_id, analyte_id=analyte_id, **_get_analyte_row(individual)))
```

Comment:
It may be better to replace the field names like `participant_id` and `analyte_id` with constants. But it could be overkilled because of so many of them.

Reply:
its overkill